src/form-builder-controls.js

```diff
@@ -131,15 +262,18 @@ export default [
           field: 'fontWeight',
           config: {
             label: 'Font Weight',
+            name: 'Font Weight',
             helper: 'The weight of the text',
-            options: [{
-              value: 'normal',
-              content: 'Normal',
-            },
-            {
-              value: 'bold',
-              content: 'Bold',
-            },
+            validation: 'required',
+            options: [
+              {
+                value: 'normal',
+                content: 'Normal',
+              },
+              {
+                value: 'bold',
+                content: 'Bold',
+              },
```

Comment:
If formatting changes are required, they should be added to the `.eslintrc.js` file. A good bit of this PR is unnecessary formatting changes; please have your code editor format based on the ESLint config file.